### PR TITLE
Unit tests for sampleService

### PR DESCRIPTION
**Pull request by divyamadala30**

### CODE REVIEW

File: service/src/main/java/org/mskcc/cmo/metadb/service/SampleService.java [hunk 1]
```diff
@@ -19,7 +19,8 @@
 
     MetaDbSample getMetaDbSample(UUID metaDbSampleId) throws Exception;
 
-    MetaDbSample getMetaDbSampleByRequestAndIgoId(String requestId, SampleAlias igoId) throws Exception;
+    MetaDbSample getMetaDbSampleByRequestAndIgoSampleAlias(String requestId,
```
Comment (n1zea144) [inline service/src/main/java/org/mskcc/cmo/metadb/service/SampleService.java:22]: Can we drop Igo from the method name?

**Review by n1zea144 — APPROVED** [review]
We are probably going to want to search for a sample/history by its alias, but I don't think we need to build it in until that comes up.

File: service/src/main/java/org/mskcc/cmo/metadb/service/SampleService.java (hunk 1 above)
Comment (ao508) [reply service/src/main/java/org/mskcc/cmo/metadb/service/SampleService.java:22]: maybe `getMetaDbSampleByRequestAndIgoSampleAlias` --> `getMetaDbSampleByRequestAndAlias` ?

**Review by ao508 — CHANGES_REQUESTED** [review]
Can you add java docs to each of the Test methods explaining what they're testing for?

File: service/src/test/java/org/mskcc/cmo/metadb/service/MockDataUtils.java [hunk 2]
```diff
@@ -88,7 +88,7 @@ public MetaDbRequest extractRequestFromJsonData(String requestJson) throws Excep
     }
 
     /**
-     * Extracts a List of MetaDbSample's given a request json string;
+     * Extracts a List<MetaDbSample> given a request json string
```
Comment (ao508) [inline service/src/test/java/org/mskcc/cmo/metadb/service/MockDataUtils.java:91]: this is where the checkstyle issue is failing, i would just revert it to what it was previously

**Review by ao508 — APPROVED** [review]

**Pull request merged**